### PR TITLE
CRM-19122 Group Organization & parent default code should be the same

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
- [CRM-19122: Group Organization & parent default code should be the same](https://issues.civicrm.org/jira/browse/CRM-19122)

### CODE REVIEW

**Comment by seamuslee001** [comment]
Jenkins re test this please

File: CRM/Contact/Form/Task/SaveSearch.php [hunk 1]
```diff
@@ -242,4 +240,17 @@ public function postProcess() {
...
 
+  /**
+   * Set form defaults.
+   *
+   * return array
+   */
+  public function setDefaultValues() {
+    $defaults = array();
+    if (empty($defaults['parents'])) {
+      $defaults['parents'] = CRM_Core_BAO_Domain::getGroupId();
```
Comment (seamuslee001) [inline CRM/Contact/Form/Task/SaveSearch.php]: @eileenmcnaughton I notice here that you are setting the default as having a parent group here, I am just wondering how that works with multisite given iirc in multisite extension we are looking at default being the domain org contact not the domain group

Comment (eileenmcnaughton) [inline CRM/Contact/Form/Task/SaveSearch.php]: hmm - have I changed the behaviour? Wouldn't that still be overridden?

Comment (seamuslee001) [inline CRM/Contact/Form/Task/SaveSearch.php]: I think it might but it was just a thought,

Comment (eileenmcnaughton) [inline CRM/Contact/Form/Task/SaveSearch.php]: I just checked & this behaviour is also in CRM_Group_Form_Edit - the multisite extension overrides it.

- would be good to get this merged

**Comment by eileenmcnaughton** [comment]
@seamuslee001 thoughts?

**Comment by seamuslee001** [comment]
with your comment @eileenmcnaughton happy for merge

**Pull request merged**